fix_withOpacity.sh

```
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script replaces all occurrences of withOpacity() with withAlpha()
+# and dynamically calculates the appropriate alpha value (opacity * 255).
+
+# Find all Dart files with withOpacity
+find lib -type f -name "*.dart" | xargs grep -l "withOpacity" > files.txt
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)